package.json

```diff
@@ -51,10 +51,11 @@
   },
   "dependencies": {
     "@amplitude/ua-parser-js": "^0.7.24",
-    "@aws-sdk/client-personalize": "^3.363.0",
+    "@aws-sdk/client-personalize": "^3.391.0",
+    "@aws-sdk/client-lambda": "^3.276.0",
```

Comment:
what for is this new dependency `client-lambda` ?

Reply:
This dependency was in the snyk fix open PR.
So, I have added . if not needed can remove this .

Reply:
can you share the PR to know whether to keep or remove it